tests/foreman/api/test_activationkey.py

```diff
@@ -126,6 +130,20 @@ def test_negative_create_3(self):
                         unlimited_content_hosts=True,
                     ).create()
 
+    def test_negative_create_4(self):
+        """@Test: Create activation key providing an invalid initial name.
+        set.
+
+        @Assert: Activation key is not created
+
+        @Feature: ActivationKey
+
```

Comment:
No need to add this blank line.

For more info check: https://www.python.org/dev/peps/pep-0008/#documentation-strings

PS: I know that we have been doing this for a long time, but some time ago I read again that section and figured out that we ensured a non PEP8 standard.